.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,20 @@
-<!--- Provide a general summary of your changes in the Title above -->
+<!--- Write a general summary of your changes in the Title field above -->
 
-## Description
-<!--- Describe your changes in detail -->
+<Addresses/Closes> #<issue/pr link>. <!---Describe the original problem or feature proposal, and briefly how this PR resolves it. All PRs should be opened against an issue.-->
 
-## Motivation and Context
-<!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to the issue here. -->
-
-## How Has This Been Tested?
```

Comment:
Can we do a checklist here? Longer term we should be able to automate a lot of this with commit hooks and tools like `prettier` but for now it might be good to remind people of what they should do before sending a PR so it won't disappoint them and break when our CI tools run. Something like this:

----
- [ ] Style and lint checked with `npm run lint`
- [ ] Production version built with `npm run build`
- [ ] Tests added to verify bug fix or new functionality
- [ ] All tests passing locally with `npm test`
----